pkg/skaffold/schema/v1alpha2/config.go

```diff
@@ -85,6 +86,12 @@ type GoogleCloudBuild struct {
 	ProjectID string `yaml:"projectId"`
 }
 
+type KanikoBuild struct {
+	GCSBucket      string `yaml:"gcsBucket,omitempty"`
+	DockerfilePath string `yaml:"dockerfilePath,omitempty"`
```

Comment:
Why does Kaniko need a DockerfilePath here, isn't that specified at the Artifact level?

Reply:
Yup, sorry. I originally had kaniko as an artifact type

Reply:
removed